pluginClientSdkLib/src/main/java/com/twofortyfouram/locale/sdk/client/receiver/AbstractAsyncReceiver.java

```diff
@@ -62,6 +62,7 @@
                 new Pair<PendingResult, AsyncCallback>(pendingResult, callback)));
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
I usually try limit the scope of a suppress annotation as much as possible, so that future warnings don't get accidentally suppressed.  What about a small helper method to perform the cast, and only suppress warnings for that helper method?  For example:

        @NonNull
        @SuppressWarnings("unchecked")
        private static Pair<PendingResult, AsyncCallback> castObj(@NonNull final Object o) {
            return (Pair<PendingResult, AsyncCallback>) o;
        }

Then the Handler could call:

        final Pair<PendingResult, AsyncCallback> pair = castObj(msg.obj);